graphics/fontforge/Portfile

```diff
@@ -93,67 +46,38 @@ depends_lib \
                 port:libuninameslist \
                 port:libxml2 \
                 port:tiff \
-                port:xorg-libXi \
-                port:xorg-libXt \
-                port:xorg-libxkbui \
-                port:zlib
-
-use_autoconf    yes
+                port:zlib \
+                port:libspiro \
+                port:pango \
+                port:cairo
 
 configure.args  --mandir=${prefix}/share/man \
-                --enable-type3 \
-                --enable-devicetables \
-                --enable-gb12345 \
-                --without-python \
-                --without-freetype-bytecode \
-                --without-pango \
-                --without-cairo \
-                --x-includes=${prefix}/include \
-                --x-libraries=${prefix}/lib
+                --disable-python-scripting \
+                --disable-python-extension \
+                --without-x
 
 configure.ldflags-append \
                 -lintl
 
-# https://trac.macports.org/ticket/29840
-compiler.blacklist \
-                llvm-gcc-4.2
-
-set docdir      ${prefix}/share/doc/${name}
-destroot.args   docdir=${docdir}
-
-post-destroot {
-    xinstall -d ${destroot}${docdir}
-    xinstall -m 644 -W ${worksrcpath} \
-        AUTHORS INSTALL LICENSE README-Mac.html README-Unix.html README-unix \
-        ${destroot}${docdir}
-    # Fix permissions problem on this file
-    xinstall -m 0644 -W ${destroot}${applications_dir}/FontForge.app/Contents/Resources FontForge.icns \
-        ${destroot}${applications_dir}/FontForge.app/Contents/Resources
-}
-
-variant freetype_bytecode description {Enable support for bytecode interpreter} {
-    distfiles-append        freetype-${ftversion}${extract.suffix}:freetype
-    extract.only-append     freetype-${ftversion}${extract.suffix}
-    configure.args-delete   --without-freetype-bytecode
-    configure.args-append   --with-freetype-src=${workpath}/freetype-${ftversion}
-}
+#PYTHON_CFLAGS=`${PYTHON}-config --cflags`
+#PYTHON_LIBS=`${PYTHON}-config --ldflags`
 
 variant python26 conflicts python27 description {Enable Python support (Python 2.6)} {
```

Comment:
Why do you need python 2.6? Wouldn't it be better to simply drop support for 2.6?

Reply:
Yes, it would. I kept it, because I wanted it to be as close to the original port as possible.
But, if changing these things is ok, I'd also make the python binding for 2.7 the default port and not a variant.

Reply:
In this case just forget about compatibility with the old port. Python 2.6 should have long been removed, it just wasn't because apparently nobody took the time yet to do it. If python 3 is not compatible yet, just make 2.7 the default.

Reply:
Have you tried what I suggested in the ticket? https://trac.macports.org/ticket/43844#comment:8